### ISSUE TITLE
log_prob 的计算问题

### ISSUE THREAD

**Issue by Taskii-Lei** [hiyouga/EasyR1#374]
在 https://github.com/hiyouga/EasyR1/blob/51001bd457f4fbab0055b6a07341c4ac468b61e6/verl/utils/torch_functional.py#L67 中，计算 log_prob 的函数为：
```
def log_probs_from_logits(logits: torch.Tensor, labels: torch.Tensor) -> torch.Tensor:
    ...
    if FLAH_ATTN_CROSS_ENTROPY_LOSS_AVAILABLE:
        output = log_probs_from_logits_flash_attn(logits, labels)
    else:  # fall back to torch kernel, upcast logits to fp32
        output = F.cross_entropy(logits.float(), labels, reduction="none")

    return output.view(*batch_dim)
```

当 if 触发的时候，输出：
```
@torch.compiler.disable()
def log_probs_from_logits_flash_attn(logits: torch.Tensor, labels: torch.Tensor) -> torch.Tensor:
    output = cross_entropy_loss(logits, labels, inplace_backward=True)
    if not isinstance(output, tuple):
        raise ValueError(
            "please make sure flash-attn>=2.4.3 where cross_entropy_loss returns Tuple[losses, z_losses]."
        )

    return -output[0]
```
当else触发则返回：
```
output = F.cross_entropy(logits.float(), labels, reduction="none")

    return output.view(*batch_dim)
```
这里的正负号好像有一些问题？cross_entropy一般得到的是 loss，应该是返回非负数的？

**Comment by hiyouga** [comment]
else 的分支的确有问题，已经修复。现在 else 的代码几乎不会执行
